subt/depth2scan.py

```diff
@@ -166,9 +166,10 @@ def update(self):
         assert channel in ["depth", "scan", "rot"], channel
 
         if channel == 'depth':
-            assert self.depth.shape == (360, 640), self.depth.shape
+            #assert self.depth.shape == (360, 640), self.depth.shape
+            pass
         elif channel == 'scan':
-            assert len(self.scan) == 720, len(self.scan)
+            #assert len(self.scan) == 720, len(self.scan)
```

Comment:
Since depth2scan is not used in the proposed changes here, there need to be no changes to it.

Reply:
I agree. Shall I just remove the change a commit again or is it necessary to do something else to not demage the history?

Reply:
Just add back the "#", commit and push. It will disappear from the diff view in the PR and that is what we want.

Reply:
Err, I meant to say "just remove the #"...